src/interface/term.ml

```diff
@@ -620,19 +620,143 @@ module type Ae_Base = sig
   val _false : t
   (** The symbol for [false] *)
 
+  val neg : t -> t
+  (** Negation. *)
+
+  val _or : t list -> t
+  (** Disjunction of formulas *)
+
+  val _and : t list -> t
+  (** Disjunction of formulas *)
+
+  val imply : t -> t -> t
+  (** Implication *)
+
+  val equiv : t -> t -> t
+  (** Equivalence *)
+
+  val xor : t -> t -> t
+  (** Exclusive disjunction. *)
+
+  val ite : t -> t -> t -> t
+  (** [ite condition then_t else_t] creates a conditional branch. *)
+
+  val distinct : t list -> t
+  (** Distinct constraints on terms. *)
+
+  val neq : t -> t -> t
+  (** Disequality. *)
```

Comment:
I think we don't need this one anymore since we always use `distinct`

Reply:
Yes, I removed it.